backend/bin/deal-observer-backend.js

```diff
@@ -7,6 +7,7 @@ import { createInflux } from '../lib/telemetry.js'
 import { getChainHead, rpcRequest } from '../lib/rpc-service/service.js'
 import { fetchDealWithHighestActivatedEpoch, observeBuiltinActorEvents } from '../lib/deal-observer.js'
 import assert from 'node:assert'
+import { pieceIndexerLoopFunction } from '../lib/piece-indexer.js'
```

Comment:
```suggestion
import { pieceIndexerLoopFunction } from '../lib/piece-indexer.js'
```

Again, `Function` or `Fn` shouldn't be part of a function name, in my opinion. Can we start with a verb instead? Something like `indexPieces()`?

Reply:
I renamed the function.